CAP/gap/MethodRecord.gi

```diff
@@ -2071,7 +2075,9 @@ UniversalMorphismIntoCoimage := rec(
   cache_name := "UniversalMorphismIntoCoimage",
   universal_object_position := "Range",
   universal_type := "Colimit",
-  
+  dual_preprocessor_func := function( arg )
+      return CAP_INTERNAL_REVERSE_LISTS_IN_ARGUMENTS( CAP_INTERNAL_OPPOSITE_RECURSIVE( arg ) );
```

Comment:
The function `REVERSE_LISTS_IN_ARGUMENTS` is never used without the `OPPOSITE_RECURSIVE` function. Maybe we could just fuse the two, or create a third that does fuse the two, and add that function directly without creating a new function for each occurance of the pair?

Reply:
I created a fused version.